tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionDetailRepositoryTests.cs

```diff
@@ -105,11 +100,6 @@ await CatalogueItemEntityBuilder
 
             await SolutionEntityBuilder.Create()
                 .WithId(Solution1Id)
-                .Build()
-                .InsertAsync();
-
-            await SolutionDetailEntityBuilder.Create()
-                .WithSolutionId(Solution1Id)
                 .WithAboutUrl("AboutUrl")
                 .WithClientApplication("Native-Desktop")
                 .Build()
```

Comment:
As above, we should call `InsertAsync` instead of `InsertAndSetCurrentForSolutionAsync`.

Reply:
Replaced `InsertAndSetCurrentForSolutionAsync` with `InsertAsync`